tardis/simulation/base.py

```diff
@@ -216,74 +200,10 @@ def estimate_t_inner(
 
         return input_t_inner * luminosity_ratios**t_inner_update_exponent
 
-    @staticmethod
-    def damped_converge(value, estimated_value, damping_factor):
-        # FIXME: Should convergence strategy have its own class containing this
-        # as a method
-        return value + damping_factor * (estimated_value - value)
-
-    def _get_convergence_status(
-        self, t_rad, w, t_inner, estimated_t_rad, estimated_w, estimated_t_inner
-    ):
-        # FIXME: Move the convergence checking in its own class.
-        no_of_shells = self.model.no_of_shells
-
-        convergence_t_rad = (
-            abs(t_rad - estimated_t_rad) / estimated_t_rad
-        ).value
-        convergence_w = abs(w - estimated_w) / estimated_w
-        convergence_t_inner = (
-            abs(t_inner - estimated_t_inner) / estimated_t_inner
-        ).value
-
-        fraction_t_rad_converged = (
-            np.count_nonzero(
-                convergence_t_rad < self.convergence_strategy.t_rad.threshold
-            )
-            / no_of_shells
-        )
-
-        t_rad_converged = (
-            fraction_t_rad_converged > self.convergence_strategy.fraction
-        )
-
-        fraction_w_converged = (
-            np.count_nonzero(
-                convergence_w < self.convergence_strategy.w.threshold
-            )
-            / no_of_shells
-        )
-
-        w_converged = fraction_w_converged > self.convergence_strategy.fraction
-
-        t_inner_converged = (
-            convergence_t_inner < self.convergence_strategy.t_inner.threshold
-        )
-
-        if np.all([t_rad_converged, w_converged, t_inner_converged]):
-            hold_iterations = self.convergence_strategy.hold_iterations
-            self.consecutive_converges_count += 1
-            logger.info(
-                f"Iteration converged {self.consecutive_converges_count:d}/{(hold_iterations + 1):d} consecutive "
-                f"times."
-            )
-            # If an iteration has converged, require hold_iterations more
-            # iterations to converge before we conclude that the Simulation
-            # is converged.
-            return self.consecutive_converges_count == hold_iterations + 1
-        else:
-            self.consecutive_converges_count = 0
-            return False
-
     def advance_state(self):
         """
         Advances the state of the model and the plasma for the next
-        iteration of the simulation. Returns True if the convergence criteria
-        are met, else False.
-
-        Returns
-        -------
-            converged : bool
```

Comment:
you should still have what the function returns(or what it updates in this case)

Reply:
Fixed this!